src/java/us/kbase/mobu/GitCommit.java

```diff
@@ -0,0 +1,38 @@
+package us.kbase.mobu;
+
+import java.io.InputStream;
+import java.util.Scanner;
+
+/** The Git commit from which the SDK was built. Expects a file called "gitcommit" in the same
+ * directory as the class file which contains the commit hash.
+ * 
+ * If the file is missing, the Git commit will be replaced with an error message.
+ * @author gaprice@lbl.gov
+ *
+ */
+public class GitCommit {
+	
+	// can't really test this easily since the file must be baked into the jar or war,
+	// just test manually
+
+	/** The Git commit from which the service was built. */
+	public static final String COMMIT;
+	
+	private static final String COMMIT_FILE_NAME = "gitcommit";
+	
+	static {
+		final InputStream is = GitCommit.class.getResourceAsStream(COMMIT_FILE_NAME);
+		String commit = null;
+		if (is == null) {
```

Comment:
thanks codecov, it's so easy to review this PR with your interpolations

Reply:
You can hide the annotations in the vertical ellipsis menu

Reply:
ZOMG! Thank you, I had not seen that feature.